### ISSUE TITLE
Updates Dashboard only showing last 4 updates

### ISSUE THREAD

**Issue by 3rdStng** [adriankumpf/teslamate#750]
The recently updated "Updates" dashboard shows that I have had 5 updates since I installed and started using TeslaMate, which is correct. However only 4 are shown in the details below it.

![2020-05-24_094416](https://user-images.githubusercontent.com/24888063/82759584-3bfeb600-9da3-11ea-8293-713f9c9d94c5.jpg)

**Comment by DrMichael** [comment]
Could you run in the directory with your docker-compose.yml the follwing:

`docker-compose exec database psql -U teslamate -d teslamate -c "select * from updates;"`

And post the result?

**Comment by 3rdStng** [comment]
I didn't ever setup the docker-compose.yml file. I built individual dockers on my unRAID server and then pointed the settings to their respective locations. Everything appears to be working, but I feel like I missed a big step. None of my dashboards auto update like a friends does. 
Should the YML file be on my main TeslaMate docker?

Regardless, here is the output from that psql command.

![image](https://user-images.githubusercontent.com/24888063/82849020-ee6d7080-9eaa-11ea-9a44-4b0c070f49c7.png)

**Comment by DrMichael** [comment]
1. Are you sure, that the upper right says "Last 10 years"?
2. The missing update has a zero duration... Should not be an issue though.

Could you please run:

```
docker-compose exec database psql -U teslamate -d teslamate << EOF
SELECT
	start_date,
	end_date,
	extract(EPOCH FROM end_date - start_date) AS update_duration,
	extract(EPOCH FROM start_date - lag(start_date) OVER (ORDER BY start_date)) AS since_last_update,
	split_part(version, ' ', 1) as version
FROM
  updates
WHERE
	start_date BETWEEN '2010-05-26T10:26:38.238Z' AND '2020-05-26T10:26:38.238Z'
	AND car_id = '1'
ORDER BY
  start_date DESC;
EOF
```

**Comment by 3rdStng** [comment]
Yes. The last 10 years was the option set. However if I change this, the number of updates and what is displayed doesn't match. Within the last 7 days, I received 2020.16.2.1 and the dashboard show 1 update, but no details.

I don't believe this is a DB issue. This started after I manually updated the "Updates" dashboard in Grafana. However the output from your commands are below.

![image](https://user-images.githubusercontent.com/24888063/82924326-0687d300-9f31-11ea-8aaa-20ebca727a80.png)

![image](https://user-images.githubusercontent.com/24888063/82924341-0be51d80-9f31-11ea-95df-16c43b5ca032.png)

![image](https://user-images.githubusercontent.com/24888063/82924351-11426800-9f31-11ea-8dcc-a89de600b1c3.png)

![image](https://user-images.githubusercontent.com/24888063/82924399-20c1b100-9f31-11ea-82e3-b5392327d24b.png)

**Comment by DrMichael** [comment]
Ah, you're using the latest updates.json from github... The SQL statement in there does not hit all updates.

**Comment by Dulanic** [comment]
I'll put in a PR, it's because I mistakenly uses JOIN vs left join. It doesn't show because he had no charges or drives between 2 versions.

**Comment by DrMichael** [comment]
Great!

**Comment by Dulanic** [comment]
I say mistakenly, but I should have said I never imagined a scenario where someone would have - charges or drives between 2 updates :) That's poor insight on my behalf.

**Comment by adriankumpf** [comment]
Fixed with #762. Thanks @Dulanic!

**Comment by 3rdStng** [comment]
My Updates dashboard shows all my updates again. Thank you!
